cvat/requirements/base.txt

```diff
@@ -47,7 +47,7 @@ azure-storage-blob==12.13.0
 google-cloud-storage==1.42.0
 # --no-binary=datumaro: workaround for pip to install
 # opencv-headless instead of regular opencv, to actually run setup script
-datumaro==0.2.0 --no-binary=datumaro
+git+https://github.com/cvat-ai/datumaro@ay/labels_extension
```

Comment:
Update this requirement as the `ay/labels_extension` extension no longer exists.

Reply:
Updated. I opened a [PR](https://github.com/opencv/cvat/pull/4984) with Datumaro version update